impedance/models/circuits/circuits.py

```diff
@@ -104,7 +104,7 @@ def fit(self, frequencies, impedance, bounds=None,
                             f'dtype (currently {frequencies.dtype})')
         if not isinstance(impedance, np.ndarray):
             raise TypeError('impedance is not of type np.ndarray')
-        if impedance.dtype != np.complex:
+        if impedance.dtype != complex:
```

Comment:
Wanted to note that I think we'll get merge conflicts here with #224. They should be easy to resolve, but it's something to be aware of